docs/components/compact.md

```diff
@@ -89,15 +89,15 @@ The main risk is the **irreversible** implications of potential configuration er
 * If you merge disjoint series in multiple of blocks together, there is currently no easy way to split them back.
 * The `penalty` offline deduplication algorithm has its own limitations. Even though it has been battle-tested for quite a long time, very few issues still come up from time to time (such as [breaking rate/irate](https://github.com/thanos-io/thanos/issues/2890)). If you'd like to enable this deduplication algorithm, do so at your own risk and back up your data first!
 
-#### Enabling Vertical Compaction
+#### Vertical Compaction
 
 **NOTE:** See the ["risks" section](#vertical-compaction-risks) to understand the implications and experimental nature of this feature.
 
-You can enable vertical compaction using the hidden flag `--compact.enable-vertical-compaction`
+Vertical compaction feature is enabled by default.
```

Comment:
I think we should rewrite this doc a bit and remove any mentions of this being experimental or "risky", if this will now be default.

Also AFAIU this will be enabled always and it won't be possible to disable it, so perhaps we should specify that here.